apps/prairielearn/src/lib/features/index.ts

```diff
@@ -8,6 +8,8 @@ const featureNames = [
   // Can only be applied to courses/institutions.
   'process-questions-in-worker',
   'question-sharing',
+  // Can only be applied globally.
+  'socket-io-long-polling-only',
```

Comment:
Bikeshed: is there a better name for this? 🤷

Reply:
Seems fine to me!